ui/src/components/common/Header.js

```diff
@@ -18,8 +19,17 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { toggleHeaderDropdown } from '../../actions/actions-ui';
 
-class Header extends React.Component {
-  onTitleClick = () => {};
+export class Header extends React.Component {
+  constructor (props) {
+    super(props);
+    this.onTitleClick = this.onTitleClick.bind(this);
+    this.onUserDropdownToggle = this.onUserDropdownToggle.bind(this);
+    this.onUserDropdownSelect = this.onUserDropdownSelect.bind(this);
+    this.onLogoutUser = this.onLogoutUser.bind(this);
+  }
+  onTitleClick () {
```

Comment:
```suggestion
  onTitleClick = () => {
```

Reply:
Converting to arrow function as suggested will remove need to bind `this` to the method in the constructor